sync_events.py

```diff
@@ -0,0 +1,222 @@
+import os
+import pytz
+import glob
+from dataclasses import dataclass
+from pathlib import Path
+
+import requests
+import yaml
+from slugify import slugify
+from dateutil.parser import parse as parse_date
```

Comment:
requirements. to run this script you need a few dependencies installed. do we need to give some guidance as to how to set up the python environment locally? or since the idea is that eventually this will run as an automation, perhaps we can skip that?

Reply:
i moved into a subfolder and also added a basic requirements.txt. probably that is good enough for now.